sem1/cw/task2.2/Source.cpp

```diff
@@ -0,0 +1,58 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <cstdio>
+#include <string>
```

Comment:
Source.cpp: In function ‘int main()’:
Source.cpp:20:35: error: ‘strlen’ was not declared in this scope
  for (int i = 0; i < strlen(string) - 1; i++) {